data/kahet/aberrant ships.txt

```diff
@@ -0,0 +1,956 @@
+	gun 8.5 -30.5 "Ka'het Ravager Beam"
+	gun -25 21 "Aberrant Spiker"
+		angle -90
+	gun 25 21 "Aberrant Spiker"
+		angle 90
+	engine -11.5 42.5
+	engine 11.5 42.5
+
+	explode "tiny explosion" 8
+	explode "small explosion" 14
+	explode "medium explosion" 26
+	explode "large explosion" 18
+	"final explode" "nuke explosion"
+	"final explode" "nuke residue fast" 50
+	description "This appears to be some kind of organo-metallic entity, made from a strange mosaic of organic tissue, conventional material, and an exotic matter that defies conventional analysis."
+	description "This variant has grown a sharp, bladed prow capable of fitting more forward weapons."
+
+
+ship "Aberrant Mole" "Aberrant Mole (Disable-able)"
+	"display name" `K-IID "Mole"`
+	"uncapturable"
+
+
+ship "Aberrant Whiskers"
+	"display name" `K-IIE "Whiskers"`
+	plural "Whiskers"
+	sprite "ship/aberrant whiskers"
+	thumbnail "thumbnail/aberrant whiskers"
+	"never disabled"
+	attributes
+		category "Interceptor"
+		"cost" 5700000
+		"shields" 6200
+		"hull" 17200
+		"required crew" 1
+		"bunks" 1
+		"mass" 331
+		"drag" 2.1
+		"heat dissipation" .9
+		"fuel capacity" 300
+		"ramscoop" 0.2
+		"cargo space" 66
+		"outfit space" 393
+		"weapon capacity" 191
+		"engine capacity" 93
+		"energy capacity" 12000
+		"inscrutable" 1
+		"energy generation" 39
+		"heat generation" 32
+		"shield heat" 3.4
+		"shield generation" 0.6
+		"hull repair rate" 1.2
+		"ka'sei" 1
+		weapon
+			"blast radius" 300
+			"shield damage" 6400
+			"hull damage" 4800
+			"hit force" 2800
+	outfits
+		"Ka'het Ravager Beam" 2
+
+		"Ka'het Primary Cooling"
+		"Ka'het Shield Restorer"
+		"Ka'het MHD Generator"
+		"Ka'het Reserve Accumulator"
+
+		"Ka'het Compact Engine" 3
+		"Hyperdrive"
+
+	gun -9 -46.5 "Ka'het Ravager Beam"
+	gun 9 -46.5 "Ka'het Ravager Beam"
+	engine -12.5 27
+	engine 12.5 27
+
+	explode "tiny explosion" 8
+	explode "small explosion" 14
+	explode "medium explosion" 26
+	explode "large explosion" 18
+	"final explode" "nuke explosion"
+	"final explode" "nuke residue fast" 50
+	description "This appears to be some kind of organo-metallic entity, made from a strange mosaic of organic tissue, conventional material, and an exotic matter that defies conventional analysis."
+	description "This variant in particular appears to be growing some sort of tail-like structure, increasing its agility."
+
+
+ship "Aberrant Whiskers" "Aberrant Whiskers (Disable-able)"
+	"display name" `K-IIE "Whiskers"`
+	"uncapturable"
+
+
+ship "Aberrant Trip"
+	"display name" `K-IIF "Trip"`
+	plural "Trips"
+	sprite "ship/aberrant trip"
+	thumbnail "thumbnail/aberrant trip"
+	"never disabled"
+	attributes
+		category "Medium Warship"
+		"cost" 5700000
+		"shields" 14200
+		"hull" 16600
+		"required crew" 1
+		"bunks" 1
+		"mass" 444
+		"drag" 8.6
+		"heat dissipation" .4
+		"fuel capacity" 600
+		"ramscoop" 0.2
+		"cargo space" 77
+		"outfit space" 339
+		"weapon capacity" 185
+		"engine capacity" 201
```

Comment:
```suggestion
		"engine capacity" 42
```
Outfit and weapon space look good already, but it seems odd to give the ship 201 engine space if it's only using 42 of it.